### PR TITLE
Update guide to reflect that we don't need to manually add imports

### PR DESCRIPTION
**Pull request by zenweasel**

### CODE REVIEW

File: developer/tutorial/customization.md [hunk 1]
```diff
@@ -366,15 +358,6 @@ specified which template we would use for a "notFound". When we get to the routi
...
-Now to make this file active, we need to add an import to the `main.js` in the root `server` directory.
-
-```
-/**
- * Custom Plugins
- */
-import "/imports/plugins/custom/beesknees/register";
-```
-
 Because we have made changes to the registry, you must now `./reaction reset` for your changes to take affect.
```
Comment (aaronjudd) [inline developer/tutorial/customization.md]: Is this true? @jshimko

Comment (jshimko) [inline developer/tutorial/customization.md]: Yep.  The plugin loader will automatically detect and import...
```sh
myplugin/client/index.js
myplugin/server/index.js
myplugin/register.js
```

Comment (aaronjudd) [inline developer/tutorial/customization.md]: so you don't need to do `reaction reset` anymore? correct...

Comment (jshimko) [inline developer/tutorial/customization.md]: If a register.js doesn't exist on next run/build, Reaction removes it from the registry.  That's the expected behavior, right?

Comment (jshimko) [inline developer/tutorial/customization.md]: However, if the registry file changed, then yes, you'd need to reset.

**Comment by aaronjudd** [comment]
👍

**Pull request merged**